ci-operator/step-registry/agent-qe/baremetal/install/ove/agent-qe-baremetal-install-ove-commands.sh

```
@@ -9,9 +9,15 @@ set -o nounset
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM ERR
 
 [ -z "${AUX_HOST}" ] && { echo "\$AUX_HOST is not filled. Failing."; exit 1; }
-[ ! -f "${SHARED_DIR}/proxy-conf.sh" ] && { echo "Proxy conf file is not found. Failing."; exit 1; }
 
-source "${SHARED_DIR}/proxy-conf.sh"
+if [ "${DISCONNECTED}" == "true" ]; then
+  [ ! -f "${SHARED_DIR}/proxy-conf.sh" ] && {
+    echo "Proxy conf file is not found. Failing."
+    exit 1
+  }
+  source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)